src/Events/WishList/duck.js

```diff
@@ -129,7 +129,7 @@ function fetchWishList(_token) {
                 dispatch(receiveWishList(response.data));
             })
             .catch((error) => {
-                dispatch(failWishList(error.response.status));
+                dispatch(failWishList(error.message));
```

Comment:
You can use this trick `error && error.message` to make sure that there won't be any surprises and this won't break.